dictionary.json

```diff
@@ -18097,9 +18806,9 @@
   },
   {
     "toaq": "teo",
-    "type": "terminator",
-    "english": "end full-text quote.",
-    "gloss": "»",
+    "type": "end quote",
+    "english": "end full-text quote",
+    "gloss": "UNQUOTE",
     "short": "",
     "keywords": [],
     "notes": [],
```

Comment:
Below this: teoqkuq isn't in S IO DO order. But also I'm not sure if it should keep the IO place now that kuq has no IO place.

Reply:
@solpahi's very own [**teqga**](https://toadua.uakci.pl/##WS2pS_AHf) is binary, so I'll go ahead and cut the IO place.